src/scripts/main.js

```diff
@@ -1 +1,62 @@
+/* eslint-disable no-console */
 'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
I would recommend to move your resolvers into own functions

```
const clickPromise = new Promise(resolve => {
  document.addEventListener('click', () => {
    resolve();
  });
});


const contextMenuPromise = new Promise(resolve => {
  document.addEventListener('contextmenu', () => {
    resolve();
  });
});
```

apply Promise.race to the secondPromise

```
const secondPromise = new Promise((resolve, reject) => {
  Promise.race([clickPromise, contextMenuPromise])
    .then((value) => resolve('Second promise was resolved'));
});
```


apply Promise.all to the thirdPromise

```
const thirdPromise = new Promise(resolve => {
  Promise.all([clickPromise, contextMenuPromise])
    .then((value) => resolve('Third promise was resolved'));
});
```